lib/game_team.rb

```diff
@@ -0,0 +1,48 @@
+class GameTeam
+#
+  @@all = nil
+
+  def self.all
+    @@all
+  end
+
```

Comment:
this method could be pulled out with inheritence

Reply:
good to note!